### PR TITLE
Add nidigital system tests for channels and sites repeated capabilities

### PR DESCRIPTION
**Pull request by sbethur**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~

- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

- Add test for channels repeated capability
- Add test for sites repeated capability
- Add test for sites/pins chained repeated capability

### List issues fixed by this Pull Request below, if any.

None

### What testing has been done?

Added new tests. CI will run the tests.

### CODE REVIEW

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 1]
```diff
@@ -129,14 +157,13 @@ def test_tdr_all_channels(multi_instrument_session):
 
     multi_instrument_session.apply_tdr_offsets(applied_offsets)
 
-    channels = [multi_instrument_session.get_channel_names(i) for i in
-                range(0, multi_instrument_session.channel_count)]
+    channels = multi_instrument_session.get_channel_names(range(0, multi_instrument_session.channel_count))
```
Comment (sbethur) [inline src/nidigital/system_tests/test_system_nidigital.py:160]: This change is pretty cool :)

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 2]
```diff
@@ -49,6 +49,25 @@ def test_self_calibrate(multi_instrument_session):
...
+    ch_1 = multi_instrument_session.get_channel_names(indices=1)
+    assert multi_instrument_session.pins[ch_1].vil == pytest.approx(1, abs=1e-3)
+
+
+def test_sites_rep_cap(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.sites[0, 1].disable_sites()
+    assert not multi_instrument_session.sites[0].is_site_enabled()
+    assert not multi_instrument_session.sites[1].is_site_enabled()
```
Comment (marcoskirsch) [inline src/nidigital/system_tests/test_system_nidigital.py:68]: assert they are enabled before disabling

Comment (marcoskirsch) [reply src/nidigital/system_tests/test_system_nidigital.py:68]: you can merge after fixing.

Comment (sbethur) [reply src/nidigital/system_tests/test_system_nidigital.py:82]: Fixed

**Pull request merged**